tests/tck/features/FunctionsAcceptance.feature

```diff
@@ -30,494 +30,494 @@
 
 Feature: FunctionsAcceptance
 
-@skip
```

Comment:
Why the drastic change in indentation?

Reply:
I changed the file (remove one skip), auto format. undo?